sdk/core/Azure.Core.TestFramework/src/RecordEntry.cs

```diff
@@ -186,7 +196,7 @@ private void SerializeBody(Utf8JsonWriter jsonWriter, string name, byte[] reques
                         // fallback to generic string writing
                         var memoryStream = new MemoryStream();
```

Comment:
IIRC, disposing a `MemoryStream` does next to nothing, but should we here anyway for future-proofing since it is disposable? Could use the newer `using` expression for ease (the scope works out to the same thing anyway).